src/libraries/System.Private.CoreLib/src/System/Math.cs

```diff
@@ -1476,5 +1478,182 @@ public static double ScaleB(double x, int n)
             double u = BitConverter.Int64BitsToDouble(((long)(0x3ff + n) << 52));
             return y * u;
         }
+
+        //
+        // Helpers, those methods are referenced from the JIT
```

Comment:
I went back and forth about distributing these helpers in the related types vs. centralizing them here. I think centralizing them here looks better.